### PR TITLE
Added support for specifying group term slug in bp_has_groups()

### PR DESCRIPTION
**Pull request by sjregan**

### CODE REVIEW

File: loader.php [hunk 1]
```diff
@@ -12,8 +12,8 @@
  * Plugin Name:       BP Groups Taxo
  * Plugin URI:        http://imathi.eu/2014/06/02/bp-groups-taxo/
  * Description:       Use WordPress built-in taxonomy to add tags to BuddyPress groups
- * Version:           1.0.0-beta6
- * Author:            imath
+ * Version:           1.0.1
+ * Author:            imath, sjregan
```
Comment (imath) [inline loader.php]: You should add a `Contributors:` tag instead and put your name in it if you wish.

File: bp-groups/bp-groups-tag.php [hunk 2]
```diff
@@ -110,21 +110,31 @@ public function enqueue_cssjs() {
...
 	 * @access public
 	 * @since BP Groups Taxo (1.0.0)
+	 * @since BP Groups Taxo (1.0.1) Support for group_term in query args
 	 */
 	public function parse_groups_query( $args = '' ) {
-		// Filter Groups by Tag in the Groups Administration screen.
-		if ( is_admin() ) {
-			$current_screen = get_current_screen();
-		}
+		$r = bp_parse_args( $args, array(
```
Comment (imath) [inline bp-groups/bp-groups-tag.php]: `wp_parse_args()` should be used instead as the interest of bp_parse_args is to pass a third argument with a filter name, and you're not using it there

File: bp-groups/bp-groups-tag.php [hunk 3]
```diff
@@ -110,21 +110,31 @@ public function enqueue_cssjs() {
...
+			'group_term'  => null,
+		) );
 
-		if ( ! empty( $current_screen->id ) && false !== strpos( $current_screen->id, 'bp-groups' ) && ! empty( $_GET['tag'] ) ) {
-			$slug = $_GET['tag'];
+		if ( !empty( $r['group_term' ])) {
```
Comment (imath) [inline bp-groups/bp-groups-tag.php]: code formatting (more spaces where they should be):
`( ! empty ( $r['group_term'] ) )`

**Comment by sjregan** [comment]
I have committed the requested changes, thanks.

**Pull request merged**